### DISCUSSION TITLE
Include configured invocations when no answer found in a mock.

### DISCUSSION THREAD

**Discussion by maeserichar** [mockk/mockk#871] [Ideas]
Hello!

When using mocks in non-relaxed mode, when calling a method with parameters different than the ones that where configured in the mock, it throws an exception with details of the not found invocation. 

It there any reason to not include in the `MockKException` the configured invocations/answers? 

I've been taking a look at the `MockKStub.kt` file and the change shouldn't be complex, so before starting a PR working on that I wanted to double check if this have been discussed in the past or if you think that this change makes sense in this project.

Thanks!

**Comment by Raibaz** [comment]
I don't remember any discussions about this in the past, and I think it makes sense to show the list of stubbed invocations when throwing an exception for not finding a stub.

Thanks for thinking about this!
